### ISSUE TITLE
CORS policy Prometheus exporter plugin for Discourse

### ISSUE THREAD

**Issue by lbouquinUniv** [bitnami/bitnami-docker-discourse#144]
**Description**

**Steps to reproduce the issue:**

1. get prometheus plugin https://github.com/discourse/discourse-prometheus in mount plugin folder
2. restarts all container 
3. make sure that prometheus is installed:  your-discourse-website.fr/metrics ( must return stats ) 
4. add in your gafana new datasource prometehus with  your-discourse-website.fr/metrics in  url  and acces browser.  
5. Import dashboard: https://grafana.com/grafana/dashboards/3539 

**Describe the results you received:**

In braowser i can see: 
Access to XMLHttpRequest at 'https://my-site.fr/metrics/api/v1/query_range?query=avg(discourse_http_sql_duration_seconds%7Bquantile%3D%220.99%22%2Caction%3D%22show%22%2Ccontroller%3D%22topics%22%2Cjob%3D%22undefined%22%7D)%20*%201000&start=1590480960&end=1590482760&step=30&timeout=60s' from origin 'https://my-grafana.fr' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource.

I add cors origin in rest configuration ( as admin) but i same the same issue. I don't know if cors is enable. 
**Describe the results you expected:**

**Additional information you deem important (e.g. issue happens only occasionally):**

**Version**

- Output of `docker version`:

```
Client: Docker Engine - Community
 Version:           19.03.8
 API version:       1.40
 Go version:        go1.12.17
 Git commit:        afacb8b
 Built:             Wed Mar 11 01:27:04 2020
 OS/Arch:           linux/amd64
 Experimental:      false

Server: Docker Engine - Community
 Engine:
  Version:          19.03.8
  API version:      1.40 (minimum version 1.12)
  Go version:       go1.12.17
  Git commit:       afacb8b
  Built:            Wed Mar 11 01:25:42 2020
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          1.2.13
  GitCommit:        7ad184331fa3e55e52b890ea95e65ba581ae3429
 runc:
  Version:          1.0.0-rc10
  GitCommit:        dc9208a3303feef5b3839f4323d9beb36df0a9dd
 docker-init:
  Version:          0.18.0
  GitCommit:        fec3683
```

- Output of `docker info`:

```
Client:
 Debug Mode: false

Server:
 Containers: 61
  Running: 41
  Paused: 0
  Stopped: 20
 Images: 348
 Server Version: 19.03.8
 Storage Driver: devicemapper
  Pool Name: docker-253:8-233-pool
  Pool Blocksize: 65.54kB
  Base Device Size: 10.74GB
  Backing Filesystem: xfs
  Udev Sync Supported: true
  Data file: /dev/loop0
  Metadata file: /dev/loop1
  Data loop file: /var/lib/docker/devicemapper/devicemapper/data
  Metadata loop file: /var/lib/docker/devicemapper/devicemapper/metadata
  Data Space Used: 55.86GB
  Data Space Total: 107.4GB
  Data Space Available: 3.221GB
  Metadata Space Used: 106.4MB
  Metadata Space Total: 2.147GB
  Metadata Space Available: 2.041GB
  Thin Pool Minimum Free Space: 10.74GB
  Deferred Removal Enabled: true
  Deferred Deletion Enabled: true
  Deferred Deleted Device Count: 0
  Library Version: 1.02.158-RHEL7 (2019-05-13)
 Logging Driver: json-file
 Cgroup Driver: cgroupfs
 Plugins:
  Volume: local
  Network: bridge host ipvlan macvlan null overlay
  Log: awslogs fluentd gcplogs gelf journald json-file local logentries splunk syslog
 Swarm: inactive
 Runtimes: runc
 Default Runtime: runc
 Init Binary: docker-init
 containerd version: 7ad184331fa3e55e52b890ea95e65ba581ae3429
 runc version: dc9208a3303feef5b3839f4323d9beb36df0a9dd
 init version: fec3683
 Security Options:
  seccomp
   Profile: default
 Kernel Version: 3.10.0-1062.18.1.el7.x86_64
 Operating System: CentOS Linux 7 (Core)
 OSType: linux
 Architecture: x86_64
 CPUs: 4
 Total Memory: 15.49GiB
 Name: v-dockertest01
 ID: VA2H:6CX4:TT4W:KZAJ:EVZQ:YCXQ:M3JG:7U24:OJGE:XOOV:N5W3:2JRL
 Docker Root Dir: /var/lib/docker
 Debug Mode: false
 Registry: https://index.docker.io/v1/
 Labels:
 Experimental: false
 Insecure Registries:
  127.0.0.0/8
 Live Restore Enabled: true

WARNING: the devicemapper storage-driver is deprecated, and will be removed in a future release.
WARNING: devicemapper: usage of loopback devices is strongly discouraged for production use.
         Use `--storage-opt dm.thinpooldev` to specify a custom block storage device.
```

- Output of `docker-compose version` (if applicable):

```
docker-compose version 1.22.0, build f46880fe
docker-py version: 3.4.1
CPython version: 3.6.6
OpenSSL version: OpenSSL 1.1.0f  25 May 2017

```

**Additional environment details (AWS, VirtualBox, Docker for MAC, physical, etc.):**
We use Traefik to redirect in correspondant docker service. Our server is used only for docker.

**Comment by marcosbc** [comment]
That is configurable in the Discourse application admin panel, see:

![Screen Shot 2020-05-27 at 16 17 57](https://user-images.githubusercontent.com/5102866/83032109-b8a8c300-a035-11ea-9d73-86637247e138.png)

The last form also links to this page, which describes the process of configuring it: https://meta.discourse.org/t/mitigate-xss-attacks-with-content-security-policy/104243

**We recommend you back up your containers before changing the value, as it could leave your site inaccessible if set wrongly.**
